pyresample/test/test_kd_tree.py

```diff
@@ -902,7 +902,7 @@ def test_nearest_area_2d_to_area_1n_no_roi(self):
         self.assertIsInstance(res.data, da.Array)
         res = res.values
         cross_sum = np.nansum(res)
-        expected = 32114793.0
+        expected = 87281406.0
```

Comment:
I realize that this looks fishy, but I tested it with real ABI data and I think it is accurate. The source area definition whose geocentric resolution is being used is 10x50 pixels (exactly what this PR is addressing). When I took that definition, moved it over to North America and tested it with Full Disk ABI data with this PR it fully covered the output. When I did it with the old resolution it had missing pixels.